src/Bicep.Cli.IntegrationTests/.editorconfig

```diff
@@ -0,0 +1,5 @@
+#### .NET Coding Conventions ####
+[*.{cs,csx,cake,vb}]
+
+# Possible multiple enumerations of IEnumerable collection
+dotnet_diagnostic.CA1851.severity = suggestion
```

Comment:
What about suppressing these globally by configuring a conditional override in all test `.csproj` files? That feels simpler to maintain than duplicating `.editorconfig` files

Reply:
Unfortunately, `NoWarn` [does not work for code analysis warnings](https://learn.microsoft.com/en-us/visualstudio/ide/how-to-suppress-compiler-warnings?view=vs-2022#suppress-code-analysis-warnings) (the `CA` category) and the only way to suppress them is to use `.editorconfig`, `.globalconfig`, or rulesets. However, this reminded me that we can override the rule for all test projects by using globs in the root `.editorconfig` file. Not sure why I didn't think about this approach earlier 😑. Pushed a commit to address this.